tests/testthat/test-sprintf_linter.R

```diff
@@ -23,7 +22,10 @@ patrick::with_parameters_test_that(
     linter <- sprintf_linter()
     unused_arg_msg <- if (getRversion() >= "4.1.0") "one argument not used by format" else NULL
 
-    expect_lint(paste0(call_name, "('hello', 1)"), unused_arg_msg, linter)
+    expect_lint(paste0(call_name, "('hello', 1)"), "constant", linter)
+
+    expect_lint(paste0(call_name, "('hello')"), "constant", linter)
+    expect_lint(paste0(call_name, "('%%')"), "constant", linter)
```

Comment:
let's add more examples with more complicated escapes like

```
%%s
%%%
%%%s
%%%%
%%%%s
```

What should be the behavior for each of these?